hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieCompactionConfig.java

```diff
@@ -249,6 +249,18 @@
           + "record size estimate compute dynamically based on commit metadata. "
           + " This is critical in computing the insert parallelism and bin-packing inserts into small files.");
 
+  public static final ConfigProperty<String> ARCHIVE_FILES_TO_KEEP_PROP = ConfigProperty
+      .key("hoodie.keep.archive.files")
+      .defaultValue("10")
+      .withDocumentation("The numbers of kept archive files under archived");
+
+  public static final ConfigProperty<String> CLEAN_ARCHIVE_FILE_ENABLE_DROP = ConfigProperty
+      .key("hoodie.archive.clean.enable")
```

Comment:
this confuses me with regular clean. Can we call it as  "hoodie.auto.trim.archive.files" or "hoodie.auto.delete.archive.files"  or something on that end.  
and and "hoodie.max.archive.files" for the previous config.

Reply:
Tanks for your review. Changed as `hoodie.auto.trim.archive.files` and `hoodie.max.archive.files`